src/app/productConfig/productConfigForm/productConfigForm.tpl.html

```diff
@@ -214,7 +269,7 @@ <h4 class="panel-title border-bottom-small mt0 pt-" translate>
         </div><!-- // panel-body -->
       </div><!-- // panel -->
     </div>
-    <div class="panel panel-default give-modal-panel">
+    <div ng-if="$ctrl.useV3 !== 'true'" class="panel panel-default give-modal-panel">
```

Comment:
The latest [EP-2525](https://jira.cru.org/browse/EP-2525) requirements involve this coming back (always present, optional).